amp/TEMPLATE/reampv2/src/modules/geocoder/components/layout/table/ActivityTable.jsx

```diff
@@ -87,51 +95,36 @@ class ActivityTable extends Component {
                     return { width: "15%" };
                 },
                 sort:true
-            },
-            {
-                dataField: "actions",
-                text: "Actions",
-                formatExtraData: this.state.selectedRowAction,
-                formatter: (cell, row, rowIndex, formatExtraData) => {
-                    console.log(formatExtraData); // nothing happens when
-                    return (
-                        <TableActions
-                            isOpen={formatExtraData === row.id ? true : false}
-                            onActionClick={this.handleActionsClick}
-                            rowId={row.id}
-                        />
-                    );
-                },
-                headerStyle: () => {
-                    return { width: "5%" };
-                },
             }
         ];
 
-        return (
-            <div className="activity-table">
-                <BootstrapTable
-                    keyField="id"
-                    scrollY
-                    data={this.props.activities}
-                    maxHeight="200px"
-                    columns={columns}
-                    classes="table-striped"
-                    expandRow={expandRow}
-                    selectRow={selectRow}
-                    pagination={paginationFactory(options)}
-                />
-            </div>
-        );
+        return this.props.activitiesPending ? (<div className="activity-table"><Loading/></div>)
+                : (<div className="activity-table">
```

Comment:
where we are duplicating <div className="activity-table"> it would be good to have it once